src/goto-programs/class_hierarchy.h

```diff
@@ -24,7 +25,7 @@ Date: April 2016
 class class_hierarchyt
 {
 public:
-  typedef std::vector<irep_idt> idst;
+  typedef std::set<irep_idt> idst;
```

Comment:
How does using a set help? Because the ordering is defined by the ordering of `irep_id` but isn't this still dependent on loading (since the class names will be given an index based on when first seen)? In any case, update commit message to explain why this fixes the problem